features/270-next-skycli/api.md

```diff
@@ -113,27 +113,137 @@ access skycli instead of using user account.
 
 ## Config files structure
 
-### ~/.skycli/skyclirc
+### ${XDG_CONFIG_HOME:-$HOME/.config}/skycli/config
```

Comment:
`XDG` stands for?

Reply:
https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html @kaojohnny

Reply:
lol KDE thing. Thanks.